lib/broadside/deploy/ecs_deploy.rb

```diff
@@ -73,6 +73,20 @@ def bootstrap
         info "Service '#{family}' doesn't exist, creating..."
         EcsManager.create_service(config.ecs.cluster, family, @deploy_config.service_config)
       end
+
+      run_bootstrap
+    end
+
+    def run_bootstrap
```

Comment:
Ah sure. Do you think we still need to call `EcsManager.deregister_last_n_tasks_definitions`?